.taskcluster.yml

```diff
@@ -142,17 +142,67 @@ tasks:
               - "git clone --recursive --quiet ${repository} &&
                  cd rust-code-analysis &&
                  git -c advice.detachedHead=false checkout ${head_rev} &&
-                 cargo build --release &&
+                 cargo build --all --release &&
                  cd target/release &&
-                 tar -zvcf /build.tar.gz --transform 's,^,rust-code-analysis-linux-x86_64/,' rust-code-analysis"
+                 strip rust-code-analysis-cli &&
+                 tar -zvcf /build.tar.gz \
+                     --transform 's,^,rust-code-analysis-linux-x86_64/,' \
+                     rust-code-analysis-cli"
             artifacts:
               public/rust-code-analysis-linux-x86_64.tar.gz:
                 expires: {$fromNow: '2 weeks'}
                 path: /build.tar.gz
                 type: file
           metadata:
-            name: rust-code-analysis release build
-            description: rust-code-analysis release build
+            name: rust-code-analysis linux release build
+            description: rust-code-analysis linux release build
+            owner: cdenizet@mozilla.com
+            source: ${repository}/raw/${head_rev}/.taskcluster.yml
+
+      - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
+        then:
+          taskId: {$eval: as_slugid("build_windows_release")}
+          dependencies:
+            - {$eval: as_slugid("lint_test_task")}
+            - {$eval: as_slugid("windows_test_task")}
+          created: {$fromNow: ''}
+          deadline: {$fromNow: '2 hour'}
+          provisionerId: proj-relman
+          workerType: win2012r2
+          payload:
+            maxRunTime: 3600
+            image: "rust:buster"
+            env:
+              PROJECT_NAME: rust-code-analysis
+            command:
+              - set VSWHERE=%programfiles(x86)%\Microsoft Visual Studio\Installer
+              - set VS_BINS=VC\Tools\MSVC\*\bin\Hostx64\x64
+              - set RUSTUP_INIT_PATH=%CD%\rustup-init
+              - set CARGO_PATH=%USERPROFILE%\.cargo\bin
+              - set PATH=%RUSTUP_INIT_PATH%;%CARGO_PATH%;%VSWHERE%;%PATH%
+              - set VSWHERE_CMD=vswhere -latest -products * -find %VS_BINS%
+              - FOR /F "tokens=*" %%o IN ('%VSWHERE_CMD%') do (SET LIB_PATH=%%o)
+              - set PATH=%LIB_PATH%;%PATH%
+              - rustup-init -yv --default-toolchain stable ^
+                                --default-host x86_64-pc-windows-msvc
+              - git clone --recursive --quiet ${repository}
+              - cd rust-code-analysis
+              - git -c advice.detachedHead=false checkout ${head_rev}
+              - cargo build --all --release
+              - cd target/release
+              - move rust-code-analysis-cli.exe C:\Users
```

Comment:
I don't know if I can run this command without admin privileges. I don't know which folder should be used to avoid this inconvenient.